packages/corelib/src/dataModel/UserEditingDefinitions.ts

```diff
@@ -21,8 +31,11 @@ export interface CoreUserEditingDefinitionForm {
 	id: string
 	/** Label to show to the user for this operation */
 	label: ITranslatableMessage
-	/** The json schema describing the form to display */
-	schema: JSONBlob<JSONSchema>
+	/** Used to group the schemas and filter them */
+	grouping?: UserEditingGroupingType[]
```

Comment:
See my previous comment as to whether this property is needed when we can add additional fields to the JSONSchema in the format of `"ui:field"`